docs/sources/reference/run.md

```diff
@@ -873,18 +873,26 @@ this driver.
 Default logging driver for Docker. Writes JSON messages to file. `docker logs`
 command is available only for this logging driver
 
+The following logging options are supported for this logging driver: [none]
+
 #### Logging driver: syslog
 
 Syslog logging driver for Docker. Writes log messages to syslog. `docker logs`
 command is not available for this logging driver
 
+The following logging options are supported for this logging driver:
+
+    --log-opt address=[tcp|udp]://host:port
+
+`address` specifies the remote syslog server address where the driver connects to.
+If not specified it defaults to the local unix socket of the running system. If
```

Comment:
Potentially yes, cause `syslog.Dial` accepts `network`(s) specified by `net.Dial` https://golang.org/pkg/net/#Dial but I'm restricting this to only `tcp` or `udp`. I'm not sure about allowing `unix` socket here too